app/models/links/ezproxy.rb

```diff
@@ -53,7 +53,7 @@ def apply_sul_proxy_prefix?
     end
 
     def libraries
-      @libraries ||= document.holdings.libraries.map(&:code)
+      @libraries ||= document.folio_holdings.map { |h| h.effective_location.library.code }
```

Comment:
If these links are rendered on the search results page, I think we want to avoid digging into `folio_holdings` for performance reasons, right?

I think we use `holdings_library_code_ssim` and/or `library_code_facet_ssim` for similar things?

Reply:
Makes sense. Fixed.